packages/berry-shell/sources/index.ts

```diff
@@ -557,9 +552,9 @@ export async function execute(command: string, args: Array<string> = [], {
   variables = {},
 }: Partial<UserOptions> = {}) {
   const normalizedEnv: {[key: string]: string} = {};
-  for (const key of Object.keys(env))
-    if (typeof env[key] !== `undefined`)
-      normalizedEnv[key.toUpperCase()] = env[key] as string;
+  for (const [key, value] of Object.entries(env))
+    if (typeof value !== `undefined`)
+      normalizedEnv[key] = value;
 
   if (paths.length > 0)
     normalizedEnv.PATH = normalizedEnv.PATH
```

Comment:
On windows, the `PATH` env variable is not in uppercase (`Path`). It might be a problem as `normalizedEnv.PATH` will be undefined (path will not use existing PATH).

Because env variables are case insensitive on windows, `process.env.PATH` is the same as `process.env.Path`. But after copy, only the original `normalizedEnv.Path` will be set (see nodejs/node#20605).

Furthermore, it seems that NodeJS will use the first `PATH` variable it find: https://github.com/libuv/libuv/pull/1837#issuecomment-387768328
So we can't expect having `PATH` and `Path` in the same environment 😿 

As it seems to be a special case, a solution could be to just do something like:
```js
for (const [key, value] of Object.entries(env))
    if (typeof value !== `undefined`)
      normalizedEnv[key !== `Path` ? key : `PATH`] = value;
```

Reply:
Interesting, thanks for the tip! I'll add the special case 👍